src/main/java/net/earthcomputer/clientcommands/command/arguments/ClientEntityArgumentType.java

```diff
@@ -524,25 +536,35 @@ boolean applicable(Parser parser) {
                     void apply(Parser parser) throws CommandSyntaxException {
                         parser.suggestor = (builder, playerNameSuggest) -> {
                             CommandSource.suggestIdentifiers(Registry.ENTITY_TYPE.getIds(), builder, "!");
-                            if (!parser.hasType)
+                            CommandSource.suggestIdentifiers(EntityTypeTags.getTagGroup().getTagIds(), builder, "!#");
```

Comment:
Hmm, shouldn't we be using the TagManager for this?

Reply:
I somewhat blindly copied Minecraft’s code.

Reply:
Ah, ok then